recipes/auditwheel/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: auditwheel
+  version: "1.1.0"
+
+source:
+  fn: auditwheel-1.1.0.tar.gz
+  url: https://pypi.python.org/packages/source/a/auditwheel/auditwheel-1.1.0.tar.gz
+  md5: f42ec0ed1ce5e09e5ef5d3ce4f9826e6
+
+build:
+  noarch_python: True
```

Comment:
Out of interest, how come? We don't really want somebody to be able to install auditwheel on Windows (AFAICT) so I'm curious. :smile: